gravitl/netclient
# fix(GRA-924): fix gui crashing after join

Depends on: https://github.com/gravitl/netclient/pull/180 

TESTING STEPS:
1. Open GUI, join a network by various means
2. Check that the GUI does not crash, and peers are loaded correctly
